index.js

```diff
@@ -1,5 +1,6 @@
 import { createMap, createStore } from 'nanostores'
 
+let id = a => a
```

Comment:
Let’s make more reable name

Reply:
That's the [identity function](http://www.zvon.org/other/haskell/Outputprelude/id_f.html) name from the Haskell standard Prelude library =)
I'll rename it to `identity`.